runtime/src/test/java/com/redhat/ipaas/runtime/ComponentsITCase.java

```diff
@@ -26,15 +26,15 @@
     @Test
     public void componentsListTest() {
         ListResult<Component> results = restTemplate().getForObject("/api/v1/components", ListResult.class);
-        assertThat(results.getItems()).hasSize(20);
-        assertThat(results.getTotalCount()).isEqualTo(50);
+        assertThat(results.getItems()).hasSize(2);
+        assertThat(results.getTotalCount()).isEqualTo(2);
     }
 
     @Test
     public void componentsGetTest() {
-        Component result = restTemplate().getForObject("/api/v1/components/1", Component.class);
+        Component result = restTemplate().getForObject("/api/v1/components/org.foo:twitter-mention-connector:1.0", Component.class);
```

Comment:
Component = Connector + config (of all the possible options + their description). I think this is all Camel inspired terminology and it was adopted by the UX team who made the model. We simply adopted that model.